Makefile

```diff
@@ -136,7 +136,7 @@ build:
 
 .PHONY: operator-image
 operator-image: generate
-	$(CONTAINER_RUNTIME) build -f build/Dockerfile . -t $(OPERATOR_IMG)
+	docker buildx build --push --platform linux/amd64,linux/ppc64le -f build/Dockerfile -t $(OPERATOR_IMG) .
```

Comment:
please keep the $(CONTAINER_RUNTIME) var here

Reply:
Can't build multi-arch images using podman